src/test/scala/mesosphere/marathon/api/v2/json/AppDefinitionSchemaJSONTest.scala

```diff
@@ -1,225 +0,0 @@
-package mesosphere.marathon
-package api.v2.json
-
-import mesosphere.UnitTest
-import mesosphere.marathon.test.MarathonTestHelper
-
-/**
-  * Tests that test that the given JSON is rejected by the JSON schema.
-  *
-  * Since the JSON is not representable by an AppDefinition,
-  * JSON is used directly.
-  */
-class AppDefinitionSchemaJSONTest extends UnitTest {
-
-  "AppDefinitionJsonSchema" should {
```

Comment:
Did we verify that these test cases are covered somewhere else? In our current validation logic... I just don't want us to decrease coverage :)

Reply:
My vote is we just delete these tests. They're testing that our JSON schema validates things properly and have all along been totally divorced from our RAML specification.